modules/customize-certificates-api-add-default.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/api-server.adoc
+
+[id="add-default-api-server-{context}"]
+= Add an API server default certificate
+
+To allow clients outside the cluster to validate the API server's
+certificate, you can replace the default certificate
+with one that is issued by a public or organizational CA.
+
+.Prerequisites
+
+* You must have a valid certificate and key in the PEM format.
+
+.Procedure
+
+. Create a secret that contains the certificate and key in the
+`openshift-config` namespace.
++
+----
+$ oc create secret tls <certificate> \//<1>
+     --cert=</path/to/cert.crt> \//<2>
+     --key=</path/to/cert.key> \//<3>
+     -n openshift-config
+----
+<1> `<certificate>` is the name of the secret that will contain
+the certificate.
+<2> `</path/to/cert.crt>` is the path to the certificate on your
+local file system.
+<3> `</path/to/cert.key>` is the path to the private key associated
+with this certificate.
+
+. Update the API server to reference the created secret.
++
+----
+$ oc patch apiserver cluster \
+     --type=merge -p \
+     "{\"spec\": {\"servingCerts\": {\"defaultServingCertificate\": \
```

Comment:
Could you please also use single-quotes to wrap the string here as well?

Reply:
Thanks for pointing this instance out - I've changed this to use single quotations as well. Again, I haven't found a way to get this to work with the `\` between lines, but you can copy and paste the command into the terminal and it will work.